libs/EXTERNAL/libblifparse/src/blif_parser.y

```diff
@@ -34,7 +34,7 @@
 %define api.namespace {blifparse}
 
 /* Name the parser class */
-%define parser_class_name {Parser}
+%define api.parser.class {Parser}
```

Comment:
@navidjafarof As I mentioned in the VTR Industry Sync-Up, the `api.parser.class` was not added until Bison 3.3. The minimum version of Bison for VTR is currently set to 3.0; and since some users of VTR may be using older versions of Ubuntu it is not trivial to upgrade the version of Bison.

This issue is currently being tracked by PR #2529 

We plan on just suppressing the deprecation warnings for liblifparse and libsdcparse (specifically for these files) until all machines are up to date.